vendor-bin/openapi-extractor/composer.json

```diff
@@ -0,0 +1,16 @@
+{
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/nextcloud/openapi-extractor"
+		}
+	],
+	"config": {
+		"platform": {
+			"php": "8.1"
+		}
+	},
+	"require-dev": {
+		"nextcloud/openapi-extractor": "dev-main"
```

Comment:
```suggestion
		"nextcloud/openapi-extractor": "dev-main#aef9a921bebdcdc42b110a51bfb1e2ca281c307a"
```
Should be pinned

Reply:
The commited composer.lock file is the pin